php/examples/VaasExample/GetVerdictByFile.php

```diff
@@ -7,18 +7,19 @@
 
 include_once("./vendor/autoload.php");
 
+
 $authenticator = new ClientCredentialsGrantAuthenticator(
-    getenv("CLIENT_ID"),
-    getenv("CLIENT_SECRET"),
-    getenv("TOKEN_URL") ?: "https://account.gdata.de/realms/vaas-production/protocol/openid-connect/token"
+    clientId: getenv("CLIENT_ID"),
+    clientSecret: getenv("CLIENT_SECRET"),
+    tokenUrl: getenv("TOKEN_URL")
 );
 
 $vaas = (new Vaas())
     ->withAuthenticator($authenticator)
-    ->withUrl(getenv("VAAS_URL") ?? "wss://gateway.production.vaas.gdatasecurity.de")
```

Comment:
Wo wird die URL jetzt gesetzt?

Reply:
The url is part of the VaasOptions here https://github.com/GDATASoftwareAG/vaas/blob/ac30ae0fcdb8023e5ae38fd2d3f30acce8a864d9/php/src/vaas/Options/VaasOptions.php#L17 and defaults to cloud prod

Reply:
Maybe we should include an example where we also set different VaasOptions.